src/studio/src/designer/backend/Designer.csproj

```diff
@@ -93,6 +94,10 @@
     </AdditionalFiles>
   </ItemGroup>
 
+  <ItemGroup>
+    <Folder Include="Infrastructure\Authentication\" />
+  </ItemGroup>
```

Comment:
Including folders this way is usually unnecessary.

Reply:
removed